ocp_resources/serving_runtime.py

```diff
@@ -13,91 +13,149 @@ class ServingRuntime(NamespacedResource):
+
             disabled (Optional[bool]):  When set to true it disables the serving runtime.
+
             protocol_versions (Optional[List[str]]): List of protocols versions used by the serving runtime.
+
             image_pull_secrets (Optional[List[Dict[str, Any]]]): List of references to secrets used for pulling images used by this SR.
+
             grpc_endpoint (Optional[str]): Port of the gRPC endpoint.
+
             grpc_data_endpoint (Optional[str]): Port of the gRPC data endpoint.
+
             http_data_endpoint (Optional[str]): HTTP endpoint for inferencing.
+
             replicas (Optional[int]): Number of replicas of the deploymnent generated by this SR.
+
             storage_helper (Optional[Dict[str, Any]]): Configuration for this runtime's use of the storage helper.
+
             built_in_adapter Optional[Dict[str, Any]]: Configuration for the built-in adapter.
+
+            affinity (Dict[str, Any]): No field description from API; please add description
+
+            spec_annotations (Dict[str, Any]): No field description from API; please add description
+
+            spec_labels (Dict[str, Any]): No field description from API; please add description
+
+            node_selector (Dict[str, Any]): No field description from API; please add description
+
+            tolerations (List[Any]): No field description from API; please add description
+
+            volumes (List[Any]): No field description from API; please add description
+
         """
         super().__init__(**kwargs)
+
+        self.affinity = affinity
+        self.spec_annotations = spec_annotations
+        self.built_in_adapter = built_in_adapter
         self.containers = containers
-        self.supported_model_formats = supported_model_formats
-        self.multi_model = multi_model
         self.disabled = disabled
-        self.protocol_versions = protocol_versions
-        self.image_pull_secrets = image_pull_secrets
-        self.grpc_endpoint = grpc_endpoint
         self.grpc_data_endpoint = grpc_data_endpoint
+        self.grpc_endpoint = grpc_endpoint
         self.http_data_endpoint = http_data_endpoint
+        self.image_pull_secrets = image_pull_secrets
+        self.spec_labels = spec_labels
+        self.multi_model = multi_model
+        self.node_selector = node_selector
+        self.protocol_versions = protocol_versions
         self.replicas = replicas
         self.storage_helper = storage_helper
-        self.built_in_adapter = built_in_adapter
+        self.supported_model_formats = supported_model_formats
+        self.tolerations = tolerations
+        self.volumes = volumes
 
     def to_dict(self) -> None:
         super().to_dict()
+
         if not self.kind_dict and not self.yaml_file:
-            if not self.containers:
-                raise MissingRequiredArgumentError(argument="'containers'")
+            if not all([
```

Comment:
@coderabbitai please open an issue about not do `if not all()` when the required args is only one.